index.js

```diff
@@ -2,8 +2,16 @@ var fs = require('fs');
 var lame = require('lame');
 var Speaker = require('speaker');
 
-fs.createReadStream(process.argv[2])
-  .pipe(new lame.Decoder())
-  .on('format', function (format) {
-    this.pipe(new Speaker(format));
-  });
+// Check if the specified sound file exists
+var sound = process.argv[2];
+if (false == fs.existsSync(sound)) {
+  console.log('File: ' + sound + ' not found.');
```

Comment:
Also, here you could just exit, so you don't need an `else` statement below

```suggestion
  console.log('File: ' + sound + ' not found.');
  process.exit(1);
```